mapfile.c

```diff
@@ -2599,9 +2601,11 @@ int initStyle(styleObj *style)
   style->linejoinmaxsize = MS_CJC_DEFAULT_JOIN_MAXSIZE;
 
   style->numbindings = 0;
+  style->nexprbindings = 0;
   for(i=0; i<MS_STYLE_BINDING_LENGTH; i++) {
     style->bindings[i].item = NULL;
     style->bindings[i].index = -1;
+    msInitExpression(&(style->exprBindings[i]));
```

Comment:
where is this freed ?

Reply:
Thanks again. I added msFreeExpression() in freeStyle() function.